docker-compose.yml

```diff
@@ -0,0 +1,175 @@
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:?}
+      MYSQL_DATABASE: ${OPENMRS_DB_NAME:?}
+      MYSQL_USER: ${OPENMRS_DB_USERNAME:?}
+      MYSQL_PASSWORD: ${OPENMRS_DB_PASSWORD:?}
+    ports:
+      - "3306:3306"
+    volumes:
+      - "openmrsdbdata:/var/lib/mysql"
+
+  bahmni-web:
+    image: bahmniindiadistro/bahmni-web-abdm:${BAHMNI_WEB_IMAGE_TAG:?}
+    profiles: ["default","openmrs"]
+    #volumes:
+      #- "${BAHMNI_APPS_PATH:?}/ui/app/:/usr/local/apache2/htdocs/bahmni"
+      #- "${BAHMNI_APPS_PATH:?}/ui/node_modules/@bower_components/:/usr/local/apache2/htdocs/bahmni/components"
+      #- "${BAHMNI_CONFIG_PATH:?}/:/usr/local/apache2/htdocs/bahmni_config/"
+
+  db:
+    container_name: db
+    image: "postgres:latest"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=welcome
+    ports:
+      - "5433:5432"
+    volumes:
+      - postgresql:/var/lib/postgresql
+      - postgresql_data:/var/lib/postgresql/data
+  rabbitmq:
+      image: "rabbitmq:3"
+      container_name: rabbitmq
+      ports:
+        - "5672:5672"
+  otp:
+      image: "bahmniindiadistro/otp-service"
+      container_name: otp
+      depends_on:
+        - db
+      environment:
+        whitelisted:numbers: ""
+
+  hiu-db-setup:
+      image: projecteka/hiu-db-initializer
+      container_name: hiu-db-setup
+      environment:
+        - 'JAVA_TOOL_OPTIONS=-Djdbc.url=jdbc:postgresql://db:5432/health_information_user -Djdbc.username=postgres -Djdbc.password=welcome'
+
+  hip:
+    container_name: hip
+    image: bahmniindiadistro/hip
+    ports:
+      - "8000:80"
+    depends_on:
+      - db
+      - otp
+      - rabbitmq
+    volumes:
+      - ./:/shared
+    environment:
+      Gateway:clientSecret: ${HIP_SECRET:?}
+      Gateway:clientId: ${HIP_CLIENTID:?}
+      Gateway:url: ${GATEWAY_URL:?}
+      OpenMrs:url: ${OPENMRS_URL:?}
+      OpenMrs:password: ${OPENMRS_PASSWORD:?}
+      Gateway:cmSuffix: ${CONSENT_MANAGEMENT_SUFFIX:?}
+      OtpService:BaseUrl: http://otp
+      hip:url: http://hip:8000
+      HEALTH_CHECK_DURATION: 30000
+      Cors:allowedOrigins: ${CORS_ALLOWED_ORIGIN:?}
+      Hip-Service:Url: ${HIP_URL:?}
+      Bahmni:Id: ${HIP_ID:?}
+
+  hiu:
+    image: bahmniindiadistro/hiu
+    container_name: hiu
+    volumes:
+      - ./:/shared
+    environment:
+      HIU_ID: ${HIU_ID:?}
+      HIU_NAME: ${HIU_NAME:?}
+      IDENTITY_JWK_URL: ${IDENTITY_JWK_URL:?}
+      DATA_PUSH_URL: ${DATA_PUSH_URL:?}
+      CONSENT_NOTIFICATION_URL: ${CONSENT_NOTIFICATION_URL:?}
+      CONSENT_MANAGEMENT_SUFFIX:  ${CONSENT_MANAGEMENT_SUFFIX:?}
+      CONSENT_MANAGEMENT_URL: ${CONSENT_MANAGEMENT_URL:?}
+      DEFAULT_PAGE_SIZE: 20
+      MAX_PAGE_SIZE: 100
+      LOCAL_STORAGE_PATH: /tmp/
+      HIU_CLIENT_ID: ${HIU_CLIENT_ID:?}
+      HIU_CLIENT_SECRET: ${HIU_CLIENT_SECRET:?}
+      POSTGRES_HOST: db
+      POSTGRES_PORT: 5432
+      CONSENT_MANAGER_DB_NAME: health_information_user
+      POSTGRES_USER: postgres
```

Comment:
this should also be in .env file?

Reply:
Modified all the hardcoded values to pick up from .env file